src/components/WorkOrders/MobileWorkingCloseWorkOrderForm.tsx

```diff
@@ -104,6 +104,7 @@ const MobileWorkingCloseWorkOrderForm = ({
   // Show validation error immediately for quick feedback
   useEffect(() => {
     trigger('workOrderFileUpload')
+    trigger('followOnFileUpload')
```

Comment:
This may have been the cause of the bug on the follow-on files - it wouldn't always re-validate when files were removed after being over the limit